README.md

```diff
@@ -1,2 +1,32 @@
 # pymath-interval
- The purpose of this Python namespace subpackage is to provide pure-python implementations of precise interval arithmetic.
+
+The purpose of this Python namespace subpackage is to provide pure-python implementations of precise interval arithmetic.
+
+Learn more about interval arithmetic on [Wikipedia](https://en.wikipedia.org/wiki/Interval_arithmetic).
+
+## Usage and Examples
+See [Usage](https://github.com/SimpleArt/pymath-interval/blob/main/USAGE.md) for walkthroughs and examples.
+
+## Installation
+
+Installing using pip:
```

Comment:
I usually show how to use this on Windows vs Linux/Mac OS like [here](https://packaging.python.org/en/latest/tutorials/packaging-projects/) at the top.